scrape/manager.go

```diff
@@ -50,8 +50,8 @@ type Manager struct {
 	append        Appendable
 	scrapeConfigs map[string]*config.ScrapeConfig
 	scrapePools   map[string]*scrapePool
-	mtx           sync.RWMutex
-	graceShut     chan struct{}
+	sync.RWMutex
```

Comment:
What is the reason for adding the functions the mutex to the public interface? A caller from the outside shouldn't be able to change the mutex state on its own. The mutex shouldn't be embedded.

Reply:
sh*t forgot about that. Reverted.